### PR TITLE
chgrp: move help strings to markdown file

### PR DESCRIPTION
**Pull request by akhilerm**
#4368 

Move help strings from `chgrp` to `chgrp.md` file

### CODE REVIEW

File: src/uu/chgrp/src/chgrp.rs [hunk 1]
```diff
@@ -10,20 +10,17 @@
 use uucore::display::Quotable;
```
Comment (sylvestre) [inline src/uu/chgrp/src/chgrp.rs:10]: please run rustfmt

File: src/uu/chgrp/src/chgrp.rs [hunk 2]
```diff
@@ -10,20 +10,17 @@
...
 use uucore::perms::{chown_base, options, IfFrom};
+use uucore::{format_usage, help_about, help_usage};
 
 use clap::{Arg, ArgAction, ArgMatches, Command};
 
 use std::fs;
 use std::os::unix::fs::MetadataExt;
 
-static ABOUT: &str = "Change the group of each FILE to GROUP.";
 static VERSION: &str = env!("CARGO_PKG_VERSION");
```
Comment (akhilerm) [inline src/uu/chgrp/src/chgrp.rs:21]: @sylvestre why is `VERSION` only present for some utils and not others.  Shouldnt we use `clap::crate_version` everywhere for uniformity?

File: src/uu/chgrp/src/chgrp.rs (hunk 1 above)
Comment (akhilerm) [reply src/uu/chgrp/src/chgrp.rs:10]: Done

File: src/uu/chgrp/src/chgrp.rs (hunk 2 above)
Comment (sylvestre) [reply src/uu/chgrp/src/chgrp.rs:21]: yeah, probably legacy, we should be uniform

Comment (sylvestre) [reply src/uu/chgrp/src/chgrp.rs:21]: don't hesitate to do it :)

**Pull request merged**